geode-core/src/test/java/org/apache/geode/internal/cache/partitioned/PutPutReplyMessageJUnitTest.java

```diff
@@ -48,7 +45,8 @@ protected void toData(OldValueImporter ovi, HeapDataOutputStream hdos) throws IO
   @Override
   protected void fromData(OldValueImporter ovi, byte[] bytes)
       throws IOException, ClassNotFoundException {
-    ((PutReplyMessage) ovi).fromData(new DataInputStream(new ByteArrayInputStream(bytes)), mock(
-        DeserializationContext.class));
+    final DataInputStream dataInputStream = new DataInputStream(new ByteArrayInputStream(bytes));
+    ((PutReplyMessage) ovi).fromData(dataInputStream,
+        InternalDataSerializer.createDeserializationContext(dataInputStream));
```

Comment:
Ah this change is necessitated by the changes in `ReplyMessage.fromData()` that causes it to use the `context` (parameter) instead of static methods on `DataSerializer`. ok ✓